evenOccurences/evenOccurences.js

```diff
@@ -10,6 +10,50 @@
  *    * console.log(onlyEven); //  4
  *    */
 
+  //
+
+  const evenOccurence = (arr) => {
+    // Your code here.
+    let ans;
+    arr.forEach((elem, i) => {
+      let result = 0;
+      // go thru array, get count of matches for element.
+      // increment result for each match
+      arr.forEach((itemFound) => {
+        if (itemFound === elem) result += 1;
+      });
+      if (result > 1 && result % 2 === 0 && ans === undefined) {  //undefined so it will stop when
```

Comment:
This is really easy for me to read, because it's how I like write my own code. But I've been told that my code sometimes isn't the most readable. haha That being said, I liked it, someone else might not. I'm referring specifically to the conditional statement on line 25 here haha. Great work!